test/integration/targets/vmware_vspan_session/tasks/main.yml

```diff
@@ -76,8 +76,8 @@
         state: "present"
         enabled: True
         description: "basic_description"
-        source_port_transmitted: 13
-        source_port_received: 13
+        source_port_transmitted: [13, 11]
```

Comment:
I would prefer you keep the existing tasks, and just extend the test playbook with the new use-case. This way we can be sure we don't introduce any regression.